src/tesla_ver/bubble_chart/bubble_chart.py

```diff
@@ -145,22 +153,14 @@ def update_dropdowns(timestamp, mdata):
 
     @app.callback(
         Output("bubble-graph-with-slider", "figure"),
-        [
-            Input("time-slider", "value"),
-            Input("y_dropdown", "value"),
-            Input("x_dropdown", "value"),
-            Input("size_dropdown", "value"),
-            Input("annotation_dropdown", "value"),
-        ],
+        [Input("time-slider", "value"), Input("y_dropdown", "value"), Input("x_dropdown", "value"),],
         [State("df-timedata", "data"), State("time-slider", "marks"), State("df-mdata", "data")],
     )
-    def update_figure(
-        time_value, y_column_name, x_column_name, size_dropdown_name, annotation_column_name, json_data, marks, mdata
-    ):
+    def update_figure(time_value, y_column_name, x_column_name, json_data, marks, mdata):
         """This callback handles updating the graph in response to user
         actions."""
         # Prevents updates without data
-        if None in [json_data, size_dropdown_name, annotation_column_name, x_column_name, y_column_name, mdata]:
+        if None in [json_data, x_column_name, y_column_name, mdata]:
             raise PreventUpdate
```

Comment:
No error message? i.e. `PreventUpdate("error message")`

Reply:
addressed in f6d5585